utils/getInvoices/tests/getInvoices.water-part.test.ts

```diff
@@ -208,9 +208,9 @@ describe('getInvoices - WATER-PART', () => {
         company,
       })
 
-      expect(invoices).not.toContainEqual({
-        type: ServiceType.WaterPart,
-      })
+      expect(invoices).toContainEqual(
+        expect.objectContaining({ type: ServiceType.WaterPart })
+      )
```

Comment:
We need to test the exact result, not only is `WaterPart` presented but also abd it's values